src/test/test_update.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import cProfile
+from pstats import Stats
+from pathlib import Path
+from unittest import TestCase, main
+from tempfile import TemporaryDirectory
+
+from update import main as update_data
+
+
+class TestUpdate(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.profiler = cProfile.Profile()
```

Comment:
It looks like you're consistently setting up and outputting this profiling behavior across different test cases. It might be worth extracting this common behavior to a parent class which extends TestCase.

Reply:
Great idea! Done.